packages/lesswrong/lib/collections/posts/schema.tsx

```diff
@@ -840,6 +840,25 @@ const schema: SchemaType<"Posts"> = {
     hidden: !isLWorAF
   },
 
+  glossary: {
+    type: Array,
+    typescriptType: "GlossaryTerm[]",
+    optional: true,
+    nullable: true,
+    blackbox: true,
+    canRead: ['guests'],
+    control: "GlossaryEditForm",
+    group: formGroups.glossary,
+    canUpdate: [userOwns, 'admins'],
```

Comment:
This isn't an actual new field on Posts, right?  If not, it doesn't need a `canUpdate`.

Reply:
(Unless we're doing something I'd strongly anti-recommend, like allowing updates to jargonTerms by way of "updating" a post's `glossary` from the client and converting that to the appropriate operation on the server, like we do for `contents` updates being converted to creating new revisions.